kate/recovery/src/index.rs

```diff
@@ -4,7 +4,11 @@ use serde::{Deserialize, Serialize};
 
 use crate::config;
 
-#[derive(Serialize, Deserialize, Debug, Clone)]
+/// Data index per application.
+/// Size is nuber of the data cells in the matrix.
+/// Index is list of pairs (app_id, start_index),
+/// where start index is index of first cell for that application.
```

Comment:
Move comments (`Size` and `Index`) into each member.

Reply:
Done